Makefile

```diff
@@ -77,6 +77,23 @@ record-nonlocal-e2e:
 		jq '[.[] | .SpecReports[]? | select(.State == "passed") | .LeafNodeText] | sort' test/e2e/report.json > ./nonlocal-e2e-specs.txt
 .PHONY: record-nonlocal-e2e
 
+e2e/local: e2e-local/setup
+	$(MAKE) e2e-local/run
+.PHONY: e2e/local
+
+e2e-local/setup:
+	source test/e2e_local_env_vars && cd demo && ./01-register-sub.sh
+.PHONY: e2e-local/setup
+
+e2e-local/run:
+	source test/e2e_local_env_vars && $(MAKE) -C test
+	source test/e2e_local_env_vars && if [ "$(DEPLOY_ENV)" = "prow" ]; then \
```

Comment:
Generally better to not have the conditional - just make 

```
ARTIFACT_DIR ?= _artifacts
JUNIT_PATH ?= $(ARTIFACT_DIR)/junit.xml
```

Then write the jUnit always, add `_artifacts` to `.gitignore` in this repo